karapace/utils.py

```diff
@@ -12,14 +12,25 @@
 from decimal import Decimal
 from http import HTTPStatus
 from kafka.client_async import BrokerConnection, KafkaClient
+from karapace.typing import JsonData
 from types import MappingProxyType
-from typing import NoReturn, overload, Union
+from typing import IO, NoReturn, Optional, overload, Union
 
-import json
+import importlib
 import kafka.client_async
 import logging
 import time
 
+ultrajson = importlib.util.find_spec("ujson")
+if ultrajson:
```

Comment:
There is no need for the global variable, is there?

```suggestion
if importlib.util.find_spec("ujson"):
```

Reply:
Done.